### PR TITLE
[pvr] fix hide no info available EPG setting

### PR DESCRIPTION
**Pull request by phunkyfish**
## Description

Setting did not function correctly if no EPG tag was available since: https://github.com/xbmc/xbmc/commit/2c3e9a46f2bfabf6c9ca192805d345d36e943067

## Types of change

- [X] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [ ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **None of the above** (please explain below)

## Checklist:

- [X] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [X] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

File: xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp [hunk 1]
```diff
@@ -307,12 +307,15 @@ namespace
     {
       if (CServiceBroker::GetPVRManager().IsParentalLocked(epgTag))
         return g_localizeStrings.Get(19266); // Parental locked
-      else if (!epgTag->Title().empty() ||
-               CServiceBroker::GetSettingsComponent()->GetSettings()->GetBool(
-                   CSettings::SETTING_EPG_HIDENOINFOAVAILABLE))
+      else if (!epgTag->Title().empty())
```
Comment (ksooo) [inline xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp:310]: This looks wrong. Now, if there is a non-empty epg tag title, the "hide" setting is ignored.

Comment (ksooo) [reply xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp:310]: Could it be that the original `|| CService` simply needs to be changed to `&& !CService...`?

Comment (phunkyfish) [reply xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp:310]: If it’s not empty there is nothing to hide.

Comment (ksooo) [reply xbmc/pvr/guilib/guiinfo/PVRGUIInfo.cpp:310]: OMG. You are ofc right.

**Review by ksooo — APPROVED** [review]
LGTM

**Comment by phunkyfish** [comment]
Jenkins build this please

**Pull request merged**